apps/omg_watcher/lib/omg_watcher/exit_processor.ex

```diff
@@ -437,6 +454,11 @@ defmodule OMG.Watcher.ExitProcessor do
 
     {:ok, state3, db_updates} = Core.finalize_in_flight_exits(state2, finalizations, invalidities)
 
+    :ok =
+      events_with_utxos
+      |> Tools.to_bus_events_data()
+      |> publish_internal_bus_events("InFlightExitIOputProcessed")
```

Comment:
What is `IOput`?

Reply:
Input or Output (credits to @jbunce) alternative is Xput 👐 

More context: we already publish `InFlightInputWithdrawn` & `InFlightOutputWithdrawn`  event from rootchain. The above event from Watcher is focused to let consumer know that particular *put can be marked as spent (if not done yet) than on the ife processing... I'm open to better names

Reply:
Okay, why not just call it `InFlightExitProcessed` if it can be either? Introducing yet another term for this isn't helpful since, both inputs and outputs are viable means to do an IFE. Also, let's consider that an `input` and `output` are references to an output.

Reply:
I can't come up with any scenario where we need this. Watcher info can do without it as all the utxos involved here are either:
* inputs of ife tx and we have an eth_event spending entry for that, as we put it into the db when exit starts
* outputs, which had to be piggybacked and we put eth events for piggybacks into the db

Reply:
@achiurizo: I'd go with `InFlightExitOutputWithdrawn` then - exit processed is fine however in my opinion it could be confused as Info-Watcher (consumer) process no exits

@pgebal: https://github.com/omisego/elixir-omg/pull/1496#discussion_r421416898